### ISSUE TITLE
RX.Clipboard does not work in iOS Safari Mobile Web

### ISSUE THREAD

**Issue by MovingGifts** [Microsoft/reactxp#891]
I noticed that the `RX.Clipboard` when used on Safari (mobile web), tested on iPhone, on a longer page, the page scrolls to the very bottom and does NOT copy the text to clipboard. Instead, you are left with the older stale text, not the freshly copied one.

It seems that the element appended to the dom in `RX.Clipboard`, is on the very bottom, and I think iOS scrolls down to it for the copy operation, but nothing is copied and the end user is left confused.

[I created this example to showing the issue](https://codesandbox.io/s/748xm7nzo0), simply open it on an iOS device in Safari to see the issue, or in the web to see the code.

Any idea how to get it to work?

**Comment by Cavitt** [comment]
Safari on iOS (and MacOS too!) have extra restrictions on copying to the clipboard.

When creating the invisible node: https://github.com/Microsoft/reactxp/blob/master/src/web/Clipboard.ts#L31

The `contentEditable` and `readOnly` attribute need to be set.

Example:
```js
function copy(el) {
    var oldContentEditable = el.contentEditable,
        oldReadOnly = el.readOnly,
        range = document.createRange();
    el.contentEditable = true;
    el.readOnly = false;
    range.selectNodeContents(el);
    var s = window.getSelection();
    s.removeAllRanges();
    s.addRange(range);
    el.setSelectionRange(0, 999999);
    el.contentEditable = oldContentEditable;
    el.readOnly = oldReadOnly;
    document.execCommand('copy');
    document.activeElement.blur();
};
```

**Comment by erictraut** [comment]
Thanks for the bug report. I just pushed a fix and published a new version of ReactXP: 1.6.0-rc.3. I forked your code sample and confirmed that it works in iOS Safari: https://codesandbox.io/s/ll6oq4p7l
